gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/DatasetInfo.java

```diff
@@ -380,65 +446,27 @@ Dataset toPb() {
   }
 
   /**
-   * Returns a builder for the DatasetInfo object given it's user-defined id.
+   * Returns a builder for a {@code DatasetInfo} object given it's identity.
```

Comment:
Nit: I feel that "id" sounds better than "identity" (applies here and also in places unmodified by this PR).